### ISSUE TITLE
Menu Bar from outside the window

### ISSUE THREAD

**Issue by grandixximo** [antonypro/QGoodWindow#8]
First of all, thank you so much for this fantastic template!

I've added a small menu bar to see the behavior of the template, in mainwindow.cpp

```
    auto menuBar = new QMenuBar;

    auto file = menuBar->addMenu("File");
    auto newfile = new QAction("New");
    file->addAction(newfile);


    QVBoxLayout *layout = new QVBoxLayout(frame);
    layout->setMargin(0);
    layout->setSpacing(0);
    layout->addWidget(title_bar);
    layout->addWidget(menuBar);
```
I've noticed some weird behavior when clicking directly on the newly created File menu bar and then clicking outside the whole window, and then back, any Idea where the problem might be?

**Comment by antonypro** [comment]
@grandixximo: Hi and thank you for appreciate this library!
The issue you reported occurs due to a incorrect widget focus handling on my code.
I'm working on a solution for this problem now.
The next version of this library will contain the solution when i have it.

**Comment by grandixximo** [comment]
@antonypro That would be great! 
FYI this bug seems Windows related, I've tested on Linux, and seems to behave ok there, it always needs two clicks on Linux to get to the menu bar, no idea on Mac, I don't have one to test on...

**Comment by antonypro** [comment]
This issue has been fixed in version 2.0. 

I'll close this issue for now, but if you believe it's not really solved, please inform me so i can reopen the issue.
